src/inmanta/server/services/environment_metrics_service.py

```diff
@@ -153,6 +153,10 @@ async def get_metric_value(
         the database. No in-memory state is being stored by this metrics collector. The provided interval
         should be interpreted as [start_interval, end_interval[
 
+        If, at the time of collection, no metric data exists for an environment and/or group,
```

Comment:
Maybe for consistency use "category" here instead of "group" ? Not a strong opinion